.circleci/config.yml

```diff
@@ -15,8 +15,16 @@ jobs:
           name: List Data Directory Contents
           command: ./scripts/run_in_ci.sh ls data/testing
 
-    #### Add your analysis here ####
-    
+      - run:
+          name: Mutect2 vs Strelka2
+          command: ./scripts/run_in_ci.sh Rscript -e "rmarkdown::render('analyses/mutect2-vs-strelka2/01-set-up.Rmd', clean = TRUE)"
```

Comment:
```suggestion
          command: ./scripts/run_in_ci.sh Rscript -e "rmarkdown::render('analyses/mutect2-vs-strelka2/01-set-up.Rmd', clean = TRUE)"
          command: ./scripts/run_in_ci.sh Rscript -e "rmarkdown::render('analyses/mutect2-vs-strelka2/02-analyze-concordance.Rmd', clean = TRUE)"
```

Reply:
This doesn't seem to work. It doesn't like having two "commands".

Reply:
I put it in the same `command` and that works.

Reply:
:today_i_learned:

Reply:
I think we want to do this:
* As individual notebooks or scripts are coming in for an analysis module, they should be added to CI as multiple runs. This validates that they can be executed at the time of review.
* Once an analysis has been reviewed and validated, the last PR for an analysis module should be a shell script used to run everything in the module. That PR also replaces the multiple runs in the config file with a single run that runs the shell script.

Reply:
I like this idea a lot 👍

Reply:
Sounds good!